xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/serverinfo/ClioServerInfo.java

```diff
@@ -0,0 +1,48 @@
+package org.xrpl.xrpl4j.model.client.serverinfo;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.google.common.annotations.Beta;
+import org.immutables.value.Value;
+
+import java.util.Optional;
+
+/**
+ * An implementation of {@link ServerInfo} that conforms to Clio server payloads.
+ */
+@Beta
+@Value.Immutable
+@JsonSerialize(as = ImmutableClioServerInfo.class)
+@JsonDeserialize(as = ImmutableClioServerInfo.class)
+public interface ClioServerInfo extends ServerInfo {
+
+  /**
+   * Construct a builder for this class.
+   *
+   * @return An {@link ImmutableClioServerInfo.Builder}.
+   */
+  static ImmutableClioServerInfo.Builder builder() {
+    return ImmutableClioServerInfo.builder();
+  }
+
+  default ServerInfoType type() {
```

Comment:
This should have `@Value.Default` I think

Reply:
Also javadoc

Reply:
Also remind me why we need this field at all?

Reply:
I agree - I only see this value used in tests, and we already have this information available via Java typing. Is there some other reason to have this?